lib/shared_data/include/SharedDataTypes.h

```diff
@@ -80,6 +88,7 @@ struct VectornavData_s
     xyz_vec<float> angular_rates;
 };
 
+/// @brief Defines errors for TC Mux to use to maintain system safety
```

Comment:
would be nice if there was an overview of what error states were caused by what here.

Reply:
me and ben havent defined these yet, will add after we begin testing